tools/devops/automation/templates/main-stage.yml

```diff
@@ -287,6 +287,7 @@ stages:
             eq(variables['Build.SourceBranch'], 'refs/heads/net7.0'),
             eq(variables['Build.SourceBranch'], 'refs/heads/net8.0'),
             eq(variables['Build.SourceBranch'], 'refs/heads/net9.0'),
+            eq(variables['Build.SourceBranch'], 'refs/heads/net10.0'),
```

Comment:
As soon as I get this out of the pipeline updating the branches should be easier.